lib/devicectl.js

```diff
@@ -246,4 +246,45 @@ export class Devicectl {
     });
     return JSON.parse(stdout).result.apps;
   }
+
+
+  /**
```

Comment:
It makes sense to do type definitions outside of the class
Maybe it even makes sense to do these in types.ts for  the general availability